src/components/Projects/HoldingsProjects.tsx

```diff
@@ -17,70 +17,67 @@ export default function HoldingsProjects() {
     theme: { colors },
   } = useContext(ThemeContext)
 
+  if (!userAddress) {
+    return (
+      <div
+        style={{
+          textAlign: 'center',
+          padding: 20,
+        }}
+        hidden={isLoading}
+      >
+        <Trans>Connect your wallet to see your holdings.</Trans>
+      </div>
+    )
+  }
+
   return (
-    <React.Fragment>
-      {!userAddress ? (
-        <div
-          style={{
-            textAlign: 'center',
-            padding: 20,
-          }}
-          hidden={isLoading}
-        >
-          <Trans>Connect your wallet to see your holdings.</Trans>
-        </div>
-      ) : (
-        <React.Fragment>
-          {projects && projects.length > 0 && (
-            <React.Fragment>
-              <Grid>
-                {projects.map(p => (
-                  <ProjectCard project={p} />
-                ))}
-              </Grid>
-            </React.Fragment>
-          )}
+    <>
+      {projects && projects.length > 0 && (
+        <Grid>
+          {projects.map(p => (
+            <ProjectCard key={`${p.id}_${p.cv}`} project={p} />
+          ))}
+        </Grid>
+      )}
 
-          {!isLoading &&
-            projects &&
-            (projects.length === 0 ? (
-              <div
-                style={{
-                  textAlign: 'center',
-                  color: colors.text.disabled,
-                  padding: 20,
-                }}
-                hidden={isLoading}
-              >
-                <Trans>You don't hold tokens for any Juicebox project.</Trans>
-              </div>
-            ) : (
-              <div
-                style={{
-                  textAlign: 'center',
-                  color: colors.text.disabled,
-                  padding: 20,
-                }}
-              >
-                {projects.length}{' '}
-                {projects.length === 1 ? t`project` : t`projects`}{' '}
-              </div>
-            ))}
+      {!isLoading &&
+        projects &&
+        (projects.length === 0 ? (
+          <div
+            style={{
+              textAlign: 'center',
+              color: colors.text.disabled,
+              padding: 20,
+            }}
+            hidden={isLoading}
+          >
+            <Trans>You don't hold tokens for any Juicebox project.</Trans>
+          </div>
+        ) : (
+          <div
+            style={{
+              textAlign: 'center',
+              color: colors.text.disabled,
+              padding: 20,
+            }}
+          >
+            {projects.length} {projects.length === 1 ? t`project` : t`projects`}{' '}
```

Comment:
can there ever be more than 1 project? If so, this should be `projects.length > 1 ?...`/`projects.length ?...`

Reply:
Not quite sure what you mean here. Here's the cases I see:

- project.length == 0 => `0 projects`
- project.length == 1 => `1 project`
- project.length == 2 => `2 projects` (and so on)

In any case, this code wasn't changed for this PR so we can handle any issues in a follow-up 🙌